internal/zero/controller/controller.go

```diff
@@ -97,3 +99,23 @@ func (c *controller) runPomeriumCore(ctx context.Context) error {
 func (c *controller) runConnect(ctx context.Context) error {
 	return c.api.Connect(ctx)
 }
+
+func (c *controller) runAnalytics() func(ctx context.Context) error {
+	disable := false
+	return func(ctx context.Context) error {
+		if disable {
```

Comment:
When would this function be invoked more than once? I see only one call to the `runFn` passed to `run()` (on line 82), but I don't see any surrounding loop. Is there some other invocation?